### DISCUSSION TITLE
Directory where downloaded songs stored

### DISCUSSION THREAD

**Discussion by JustNice** [z-huang/music#1382] [Q&A] [answered]
How can i find the directory where downloaded songs are stored?

I tried to search in Android/data, but didnt find anything

**Comment by z-huang** [comment, marked as answer]
Songs are downloaded in chunks and in `.exo` format and they are stored in app-specific directory, where you cannot access without root permission.

**Reply by JustNice** [reply]
Oh, okay, thanks.
